libfwupd/fwupd-enums.h

```diff
@@ -124,6 +126,8 @@ typedef enum {
  * @FWUPD_DEVICE_FLAG_NO_GUID_MATCHING:		Force an explicit ID match when adding devices to the device list
  * @FWUPD_DEVICE_FLAG_UPDATABLE_HIDDEN:		Device is updatable but should not be called by the client
  * @FWUPD_DEVICE_FLAG_SKIPS_RESTART:		Device relies upon activation or power cycle to load firmware
+ * @FWUPD_DEVICE_FLAG_HAS_MULTIPLE_BRANCHES:	Device has multiple branches of firmware
```

Comment:
this description should explain what a branch actually is.  IE "Device supports switching to a different stream of firmware"

Reply:
fixed.